COMPATIBILITY.md

```diff
@@ -10,3 +10,5 @@
 | 7 (Mainnet)               | v0.79.x                | v0.37.x  |   v1     |
 | 8 (Mainnet)               | v0.80.x                | v0.37.x  |   v1     |
 | 9 (Mainnet)               | v0.81.x                | v0.37.x  |   v1     |
+| 9 (Mainnet)               | v1.0.x                 | v0.37.x  |   v1     |
+| 10 (Mainnet)              | v1.1.x                 | v0.37.x  |   v1     |
```

Comment:
Wait, if 10 is breaking compared to 9, shouldn't it actually be version 2.0.x?

Reply:
Yes, we should be referring to the LQT changes at v2. I've updated the PR accordingly.

Reply:
To recap prior discussion, however, we're bumping the crate versions because of breaking changes in the Rust APIs, not because of the protocol version change, which is tracked separately. The compatibility doc was created to disambiguate here, but we should still write down a full explanation of versioning scheme and stick those in the dev docs.